datafusion/substrait/tests/testdata/tpch_queries/query_01.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+    l_returnflag,
+    l_linestatus,
+    sum(l_quantity) AS sum_qty,
+    sum(l_extendedprice) AS sum_base_price,
+    sum(l_extendedprice * (1 - l_discount)) AS sum_disc_price,
+    sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)) AS sum_charge,
+    avg(l_quantity) AS avg_qty,
+    avg(l_extendedprice) AS avg_price,
+    avg(l_discount) AS avg_disc,
+    count(*) AS count_order
+FROM
+    lineitem
+WHERE
+    l_shipdate <= CAST('1998-09-02' AS date)
+GROUP BY
+    l_returnflag,
+    l_linestatus
+ORDER BY
+    l_returnflag,
+    l_linestatus;
```

Comment:
I believe that there are already have copies of the TCPH queries in the repo: 
https://github.com/apache/datafusion/tree/main/benchmarks/queries

I think it would be better to re-use them if possible.

Reply:
It looks like the schemas are also available as well.
https://github.com/apache/datafusion/blob/405b99ce7b1ba29dd1d406f4554d84b9f756d604/benchmarks/src/tpch/mod.rs#L44-L45